README.md

```diff
@@ -0,0 +1,72 @@
+# Mainframe Browser
+
+A browser for the new decentralized web. Makes building and running distributed, unstoppable apps easier. Built with electron.
+
+## Overview
+
+The main components (packages) of the Mainframe Browser are as follows:
+
+- `daemon`: a background daemon process
+- `launcher`: a front-end browser (powered by electron)
```

Comment:
I'd like to mention it because it helps calibrate developer expectations. Electron is a well-known framework for building cross-platform desktop apps.

Reply:
We are using Electron but developers will use the sandbox. Eventually we'll also want to support mobile environments, so Electron is an implementation detail, I think it will be worth describing the tech stack at some point but I wouldn't single-out Electron like this.

Reply:
I think we may have slightly different views on who the audience is. Right now, I anticipate most of our developers will want to be able to run js-mainframe. So the audience is internal--people who want to run this platform. Eventually, I can see the readme evolving to an external audience, i.e. those who want to build apps within the sandbox. But right now, we need more devs to be able to see and run this alpha platform.